src/storm/storage/expressions/Type.h

```diff
@@ -187,7 +196,7 @@ class IntegerType : public BaseType {
     virtual bool isIntegerType() const override;
 
    private:
-    static const uint64_t mask = (1ull << 62);
+    static const uint64_t mask = (1ull << 58);
```

Comment:
Is there a reason these have changed?

Reply:
Yes, it was required for the type comparison here implemented [here](https://github.com/moves-rwth/storm/pull/556/files/af3337abf9f01100c009d5e38ee5867ceafd84b9#diff-7a2243d7193a5ddf87c896ae38a351a8daf446c1622176d0c5d9d29e1c43c3ddR301): to ensure that a transcendental number type is considered a non-integer number by the various model visitors, its mask must evaluate to a value higher than the one inn IntegerType.
Since in this implementation the mask is generated by setting a single bit in a uint64 variable, RationalType was getting the highest possible value, and IntegerType the one right before that. For this reason, I had to shift all of them to make space for the newly implemented TranscendentalNumberType.

Reply:
Thanks for clarifying. I just wanted to make sure that I didn't miss some detail concerning the unused masks.